lib/error_messages.rb

```diff
@@ -15,7 +15,13 @@ def error_message_for_404
   end
 
   def error_message_for_500
-    'Internal server error'
+    case exception
+    when Moped::Errors::ConnectionFailure
+      hosts = current_user.active_mongo_session.try(:hosts) || ['localhost:27017']
```

Comment:
Nie wiem jak to działą w Mongoidzie, ale czy to poprawne założenie, że skoro nie ma hosta to hostem jest domyślnie localhost? Może lepiej zwrócić tutaj informację w stylu `unknown host`?